### PR TITLE
Issue #13345: Enable Empty Statement Check

### PR DESCRIPTION
**Pull request by suniti0804**
Issue: #13345

### CODE REVIEW

**Comment by romani** [comment]
Single commit please, check failing CIs and fix issues reported by them.

File: src/xdocs/checks/coding/emptystatement.xml [hunk 1]
```diff
@@ -30,14 +30,16 @@
...
+  public void foo() {
+    int i = 5;
+    if(i &gt; 3); // violation
     i++;
-  for (i = 0; i &lt; 5; i++); // violation
+    for (i = 0; i &lt; 5; i++); // violation
     i++;
-  while (i &gt; 10) // OK
+    while (i &gt; 10) // OK
     i++;
```
Comment (romani) [inline src/xdocs/checks/coding/emptystatement.xml:41]: Please remove ok trailing comments, we can keep them only it has some extra text

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs/checks/coding/emptystatement.xml (hunk 1 above)
Comment (suniti0804) [reply src/xdocs/checks/coding/emptystatement.xml:41]: do we need to remove OK trailing comments from Example1.java as well?

Comment (romani) [reply src/xdocs/checks/coding/emptystatement.xml]: Yes

**Comment by romani** [comment]
@suniti0804 , please do not resolve conversations, just reply as "done" in each review thread. Maintainers will resolve as confirmation that update is done as expected

**Review by romani — APPROVED** [review]
Ok to merge

**Review by rnveach — APPROVED** [review]

**Comment by suniti0804** [comment]
@romani please add hacktoberfest tag

**Pull request merged**